spectacles/cli.py

```diff
@@ -579,7 +594,7 @@ def iter_errors(lookml: List) -> Iterable:
             )
         if mode == "batch":
             logger.info(
-                f"\n\nTo determine the exact dimensions responsible for {'this error' if project.number_of_errors == 1 else 'these errors'}, "
+                f"\n\nTo determine the exact dimensions responsible for {'this error' if len(errors) == 1 else 'these errors'}, "
```

Comment:
@DylanBaker, is this right?

Reply:
Just wanted to make sure it wasn't an artifact of the rebases because I thought that `number_of_errors` was a new feature you added?

Reply:
Yeah. I'm not sure how I missed this in my work, but because we aren't returning the project anymore, we don't have the project to call it from. So we can now just count errors.

I think your work must have been happening in parallel and I just wasn't thinking ahead enough.